contracts/root/withdrawManager/WithdrawManagerStorage.sol

```diff
@@ -17,6 +17,7 @@ contract ExitsDataStructure {
     bytes32 txHash;
     bool burnt;
     address predicate;
+    uint256 inputCount;
```

Comment:
@atvanguard if possible please restructure this `struct` 
```
struct PlasmaExit {
    uint256 receiptAmountOrNFTId;
    uint256 inputCount;
    bytes32 txHash;
    bytes32 txHash;
    address owner;
    address token;
    address predicate;
    address predicate;
    bool burnt;
    bool burnt;
    // Mapping from age of input to Input
    mapping(uint256 => Input) inputs;
  }
```
possibly it will pack our vars into multiple 32 bytes slots.